### PR TITLE
Fix some typos and add some trailing newlines

### PR DESCRIPTION
**Pull request by elliotb-lowrisc**

### CODE REVIEW

File: doc/dev/ref-manual.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This also means you can customize your design.
 You may want to have a different number of UARTs or SPI blocks for example.
 This document describes the base configuration.
 
-The FPGA image is parameterizable to enable custom setups.
+The FPGA image is parametrisable to enable custom setups.
```
Comment (marnovandermaas) [inline doc/dev/ref-manual.md:11]: I prefer the other spelling.

File: doc/ip/pinmux.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ There are output pin selectors, which select which block output is connected to
 The selector is one-hot, so you need to write `8'b100` if you want to select input 3 for example.
 The default value for all of these selectors is `'b10`.
 
-| Address | Pin output | Possible block inputs |
-|---------|------------|-----------------------|
```
Comment (marnovandermaas) [inline doc/ip/pinmux.md:14]: This is auto generated so you need to change the template.

**Review by marnovandermaas — COMMENTED** [review]
Thank you just a few comments.

**Comment by elliotb-lowrisc** [comment]
Addressed comments

File: doc/ip/pinmux.md [hunk 3]
```diff
@@ -10,8 +10,8 @@ There are output pin selectors, which select which block output is connected to
...
+| Address | Pin output | Possible block outputs |
+|---------|------------|------------------------|
```
Comment (marnovandermaas) [inline doc/ip/pinmux.md:14]: This also needs to go in the template right? Please use top gen py to render this file.

**Review by marnovandermaas — COMMENTED** [review]
One more minor comment.

File: doc/ip/pinmux.md (hunk 3 above)
Comment (elliotb-lowrisc) [reply doc/ip/pinmux.md:14]: Oops. OK, I'll have a go at figuring-out topgen

**Comment by elliotb-lowrisc** [comment]
Correct template change, regenerate using top_gen.py

**Review by marnovandermaas — APPROVED** [review]
Thanks for making these changes!

**Pull request merged**